content/blog/gatsby-tina-101.md

```diff
@@ -207,15 +209,15 @@ module.exports = {
 
 ### Friendly Form Fields
 
-Our sidebar is okay for making changes to articles, but it would be a lot nicer if the form fields weren't things like `rawFrontmatter.title`. Our content authors would likely not appreciate labels like that!
+Right now our sidebar forms for editing are 'okay,' but there's room for improvement. It would be a lot nicer if the form fields labelled with things like `rawFrontmatter.title`. Our content authors would likely not appreciate labels like that!
```

Comment:
```suggestion
Right now our sidebar forms for editing are 'okay,' but there's room for improvement. It would be a lot nicer if the form fields weren't labeled with things like `rawFrontmatter.title`. Our content authors would likely not appreciate labels like that!
```